### DISCUSSION TITLE
为什么显示繁体字？

### DISCUSSION THREAD

**Discussion by FalconWu2017** [ofajs/discussion#20] [General]
为什么这里配置的是繁体字？
![image](https://github.com/user-attachments/assets/070808be-c0f7-4a40-8ec2-a2a5a123b657)

**Comment by kirakiray** [comment]
你本地语言是 `zh-cn` 的话才会用简体中文的报错信息，如果是 `zh-` ，非 `zh-cn` 的情况，都会默认繁体中文报错。

可以在你浏览器打印台试试 `navigator.language` 看显示什么。

**Reply by FalconWu2017** [reply]
确实是zh  这个要怎么改？

**Reply by kirakiray** [reply]
> 确实是zh 这个要怎么改？

是系统本地的语言设置来的，你的 `navigator.language` 是什么？

😂 不过的确是我的失误，应该提供一种渠道让使用者主动选择用什么语言打印，我晚点更新一下这个报错打印信息的功能。

**Reply by FalconWu2017** [reply]
我的 navigator.language 就是zh

**Reply by kirakiray** [reply]
https://github.com/ofajs/ofa-errors

**Reply by kirakiray** [reply]
将 https://github.com/ofajs/ofa-errors/blob/main/dist/ofa-error-zh.js 下载到本地，并在 ofa.js 后面添加，可以解决你的问题。
